### ISSUE TITLE
Incompatibility with node v19.0.0

### ISSUE THREAD

**Issue by IngoMeyer441** [Beaglefoot/tree-sitter-awk#9]
When trying to install `tree-sitter-awk` with `npm` (`npm install tree-sitter-awk`) I get this error message:

```text
npm ERR! code 1
npm ERR! path /home/imeyer/node_test/node_modules/tree-sitter-awk
npm ERR! command failed
npm ERR! command sh -c node-gyp rebuild
npm ERR! make: Entering directory '/home/imeyer/node_test/node_modules/tree-sitter-awk/build'
npm ERR!   CXX(target) Release/obj.target/tree_sitter_awk_binding/bindings/node/binding.o
npm ERR! make: Leaving directory '/home/imeyer/node_test/node_modules/tree-sitter-awk/build'
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.3.0
npm ERR! gyp info using node@19.0.0 | linux | x64
npm ERR! gyp info find Python using Python version 3.10.8 found at "/usr/bin/python3"
npm ERR! gyp info spawn /usr/bin/python3
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/imeyer/node_test/node_modules/tree-sitter-awk/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/imeyer/.cache/node-gyp/19.0.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/home/imeyer/.cache/node-gyp/19.0.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/home/imeyer/.cache/node-gyp/19.0.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/home/imeyer/node_test/node_modules/tree-sitter-awk',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info spawn make
npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! In file included from ../../nan/nan.h:176,
npm ERR!                  from ../bindings/node/binding.cc:3:
npm ERR! ../../nan/nan_callbacks.h:55:23: error: ‘AccessorSignature’ is not a member of ‘v8’
npm ERR!    55 | typedef v8::Local<v8::AccessorSignature> Sig;
npm ERR!       |                       ^~~~~~~~~~~~~~~~~
npm ERR! ../../nan/nan_callbacks.h:55:40: error: template argument 1 is invalid
npm ERR!    55 | typedef v8::Local<v8::AccessorSignature> Sig;
npm ERR!       |                                        ^
npm ERR! ../../nan/nan.h: In function ‘void Nan::SetAccessor(v8::Local<v8::ObjectTemplate>, v8::Local<v8::String>, GetterCallback, SetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, imp::Sig)’:
npm ERR! ../../nan/nan.h:2542:19: error: no matching function for call to ‘v8::ObjectTemplate::SetAccessor(v8::Local<v8::String>&, void (*&)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Value>&), void (*&)(v8::Local<v8::Name>, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<void>&), v8::Local<v8::Object>&, v8::AccessControl&, v8::PropertyAttribute&, Nan::imp::Sig&)’
npm ERR!  2542 |   tpl->SetAccessor(
npm ERR!       |   ~~~~~~~~~~~~~~~~^
npm ERR!  2543 |       name
npm ERR!       |       ~~~~         
npm ERR!  2544 |     , getter_
npm ERR!       |     ~~~~~~~~~      
npm ERR!  2545 |     , setter_
npm ERR!       |     ~~~~~~~~~      
npm ERR!  2546 |     , obj
npm ERR!       |     ~~~~~          
npm ERR!  2547 |     , settings
npm ERR!       |     ~~~~~~~~~~     
npm ERR!  2548 |     , attribute
npm ERR!       |     ~~~~~~~~~~~    
npm ERR!  2549 |     , signature);
npm ERR!       |     ~~~~~~~~~~~~   
npm ERR! In file included from /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-function.h:15,
npm ERR!                  from /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8.h:33,
npm ERR!                  from /home/imeyer/.cache/node-gyp/19.0.0/include/node/node.h:73,
npm ERR!                  from ../bindings/node/binding.cc:2:
npm ERR! /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-template.h:807:8: note: candidate: ‘void v8::ObjectTemplate::SetAccessor(v8::Local<v8::String>, v8::AccessorGetterCallback, v8::AccessorSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, v8::SideEffectType, v8::SideEffectType)’
npm ERR!   807 |   void SetAccessor(
npm ERR!       |        ^~~~~~~~~~~
npm ERR! /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-template.h:812:22: note:   no known conversion for argument 7 from ‘Nan::imp::Sig’ {aka ‘int’} to ‘v8::SideEffectType’
npm ERR!   812 |       SideEffectType getter_side_effect_type = SideEffectType::kHasSideEffect,
npm ERR!       |       ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
npm ERR! /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-template.h:814:8: note: candidate: ‘void v8::ObjectTemplate::SetAccessor(v8::Local<v8::Name>, v8::AccessorNameGetterCallback, v8::AccessorNameSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, v8::SideEffectType, v8::SideEffectType)’
npm ERR!   814 |   void SetAccessor(
npm ERR!       |        ^~~~~~~~~~~
npm ERR! /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-template.h:819:22: note:   no known conversion for argument 7 from ‘Nan::imp::Sig’ {aka ‘int’} to ‘v8::SideEffectType’
npm ERR!   819 |       SideEffectType getter_side_effect_type = SideEffectType::kHasSideEffect,
npm ERR!       |       ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
npm ERR! ../bindings/node/binding.cc: At global scope:
npm ERR! /home/imeyer/.cache/node-gyp/19.0.0/include/node/node.h:976:7: warning: cast between incompatible function types from ‘void (*)(v8::Local<v8::Object>, v8::Local<v8::Object>)’ to ‘node::addon_register_func’ {aka ‘void (*)(v8::Local<v8::Object>, v8::Local<v8::Value>, void*)’} [-Wcast-function-type]
npm ERR!   976 |       (node::addon_register_func) (regfunc),                          \
npm ERR!       |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
npm ERR! /home/imeyer/.cache/node-gyp/19.0.0/include/node/node.h:1010:3: note: in expansion of macro ‘NODE_MODULE_X’
npm ERR!  1010 |   NODE_MODULE_X(modname, regfunc, NULL, 0)  // NOLINT (readability/null_usage)
npm ERR!       |   ^~~~~~~~~~~~~
npm ERR! ../bindings/node/binding.cc:26:1: note: in expansion of macro ‘NODE_MODULE’
npm ERR!    26 | NODE_MODULE(tree_sitter_awk_binding, Init)
npm ERR!       | ^~~~~~~~~~~
npm ERR! make: *** [tree_sitter_awk_binding.target.mk:120: Release/obj.target/tree_sitter_awk_binding/bindings/node/binding.o] Error 1
npm ERR! gyp ERR! build error 
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:203:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:293:12)
npm ERR! gyp ERR! System Linux 5.15.76-1-MANJARO
npm ERR! gyp ERR! command "/usr/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd /home/imeyer/node_test/node_modules/tree-sitter-awk
npm ERR! gyp ERR! node -v v19.0.0
npm ERR! gyp ERR! node-gyp -v v9.3.0
npm ERR! gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/imeyer/.npm/_logs/2022-11-14T09_46_47_254Z-debug-0.log
```

This is the complete log:

```
0 verbose cli /usr/bin/node /usr/local/bin/npm
1 info using npm@9.1.1
2 info using node@v19.0.0
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 3ms
5 timing config:load:file:/usr/local/lib/node_modules/npm/npmrc Completed in 2ms
6 timing config:load:builtin Completed in 2ms
7 timing config:load:cli Completed in 2ms
8 timing config:load:env Completed in 1ms
9 timing config:load:file:/home/imeyer/node_test/.npmrc Completed in 0ms
10 timing config:load:project Completed in 1ms
11 timing config:load:file:/home/imeyer/.npmrc Completed in 0ms
12 timing config:load:user Completed in 0ms
13 timing config:load:file:/usr/etc/npmrc Completed in 1ms
14 timing config:load:global Completed in 1ms
15 timing config:load:setEnvs Completed in 1ms
16 timing config:load Completed in 13ms
17 timing npm:load:configload Completed in 13ms
18 timing npm:load:mkdirpcache Completed in 0ms
19 timing npm:load:mkdirplogs Completed in 1ms
20 verbose title npm install tree-sitter-awk
21 verbose argv "install" "tree-sitter-awk"
22 timing npm:load:setTitle Completed in 1ms
23 timing config:load:flatten Completed in 4ms
24 timing npm:load:display Completed in 6ms
25 verbose logfile logs-max:10 dir:/home/imeyer/.npm/_logs/2022-11-14T09_42_44_512Z-
26 verbose logfile /home/imeyer/.npm/_logs/2022-11-14T09_42_44_512Z-debug-0.log
27 timing npm:load:logFile Completed in 5ms
28 timing npm:load:timers Completed in 0ms
29 timing npm:load:configScope Completed in 0ms
30 timing npm:load Completed in 27ms
31 silly logfile start cleaning logs, removing 1 files
32 timing config:load:flatten Completed in 0ms
33 timing arborist:ctor Completed in 1ms
34 silly logfile done cleaning log files
35 timing arborist:ctor Completed in 0ms
36 timing idealTree:init Completed in 13ms
37 timing idealTree:userRequests Completed in 5ms
38 silly idealTree buildDeps
39 silly fetch manifest tree-sitter-awk@*
40 http fetch GET 200 https://registry.npmjs.org/tree-sitter-awk 821ms (cache miss)
41 silly placeDep ROOT tree-sitter-awk@0.5.1 OK for:  want: *
42 silly fetch manifest nan@2.14.2
43 http fetch GET 200 https://registry.npmjs.org/nan 55ms (cache updated)
44 timing idealTree:#root Completed in 893ms
45 silly placeDep ROOT nan@2.14.2 OK for: tree-sitter-awk@0.5.1 want: 2.14.2
46 timing idealTree:node_modules/tree-sitter-awk Completed in 2ms
47 timing idealTree:node_modules/nan Completed in 0ms
48 timing idealTree:buildDeps Completed in 897ms
49 timing idealTree:fixDepFlags Completed in 1ms
50 timing idealTree Completed in 918ms
51 timing reify:loadTrees Completed in 920ms
52 timing reify:diffTrees Completed in 1ms
53 silly reify moves {}
54 timing reify:retireShallow Completed in 1ms
55 timing reify:createSparse Completed in 2ms
56 timing reify:loadBundles Completed in 0ms
57 silly tarball no local data for tree-sitter-awk@https://registry.npmjs.org/tree-sitter-awk/-/tree-sitter-awk-0.5.1.tgz. Extracting by manifest.
58 timing reifyNode:node_modules/nan Completed in 43ms
59 http fetch GET 200 https://registry.npmjs.org/tree-sitter-awk/-/tree-sitter-awk-0.5.1.tgz 1324ms (cache miss)
60 timing reifyNode:node_modules/tree-sitter-awk Completed in 1333ms
61 timing reify:unpack Completed in 1333ms
62 timing reify:unretire Completed in 0ms
63 timing build:queue Completed in 2ms
64 info run tree-sitter-awk@0.5.1 install node_modules/tree-sitter-awk node-gyp rebuild
65 info run tree-sitter-awk@0.5.1 install { code: 1, signal: null }
66 timing reify:rollback:createSparse Completed in 5ms
67 timing reify:rollback:retireShallow Completed in 0ms
68 timing command:install Completed in 5556ms
69 verbose stack Error: command failed
69 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/lib/index.js:53:27)
69 verbose stack     at ChildProcess.emit (node:events:513:28)
69 verbose stack     at maybeClose (node:internal/child_process:1098:16)
69 verbose stack     at ChildProcess._handle.onexit (node:internal/child_process:304:5)
70 verbose pkgid tree-sitter-awk@0.5.1
71 verbose cwd /home/imeyer/node_test
72 verbose Linux 5.15.76-1-MANJARO
73 verbose node v19.0.0
74 verbose npm  v9.1.1
75 error code 1
76 error path /home/imeyer/node_test/node_modules/tree-sitter-awk
77 error command failed
78 error command sh -c node-gyp rebuild
79 error make: Entering directory '/home/imeyer/node_test/node_modules/tree-sitter-awk/build'
79 error   CXX(target) Release/obj.target/tree_sitter_awk_binding/bindings/node/binding.o
79 error make: Leaving directory '/home/imeyer/node_test/node_modules/tree-sitter-awk/build'
80 error gyp info it worked if it ends with ok
80 error gyp info using node-gyp@9.3.0
80 error gyp info using node@19.0.0 | linux | x64
80 error gyp info find Python using Python version 3.10.8 found at "/usr/bin/python3"
80 error gyp http GET https://nodejs.org/download/release/v19.0.0/node-v19.0.0-headers.tar.gz
80 error gyp http 200 https://nodejs.org/download/release/v19.0.0/node-v19.0.0-headers.tar.gz
80 error gyp http GET https://nodejs.org/download/release/v19.0.0/SHASUMS256.txt
80 error gyp http 200 https://nodejs.org/download/release/v19.0.0/SHASUMS256.txt
80 error gyp info spawn /usr/bin/python3
80 error gyp info spawn args [
80 error gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
80 error gyp info spawn args   'binding.gyp',
80 error gyp info spawn args   '-f',
80 error gyp info spawn args   'make',
80 error gyp info spawn args   '-I',
80 error gyp info spawn args   '/home/imeyer/node_test/node_modules/tree-sitter-awk/build/config.gypi',
80 error gyp info spawn args   '-I',
80 error gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
80 error gyp info spawn args   '-I',
80 error gyp info spawn args   '/home/imeyer/.cache/node-gyp/19.0.0/include/node/common.gypi',
80 error gyp info spawn args   '-Dlibrary=shared_library',
80 error gyp info spawn args   '-Dvisibility=default',
80 error gyp info spawn args   '-Dnode_root_dir=/home/imeyer/.cache/node-gyp/19.0.0',
80 error gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/npm/node_modules/node-gyp',
80 error gyp info spawn args   '-Dnode_lib_file=/home/imeyer/.cache/node-gyp/19.0.0/<(target_arch)/node.lib',
80 error gyp info spawn args   '-Dmodule_root_dir=/home/imeyer/node_test/node_modules/tree-sitter-awk',
80 error gyp info spawn args   '-Dnode_engine=v8',
80 error gyp info spawn args   '--depth=.',
80 error gyp info spawn args   '--no-parallel',
80 error gyp info spawn args   '--generator-output',
80 error gyp info spawn args   'build',
80 error gyp info spawn args   '-Goutput_dir=.'
80 error gyp info spawn args ]
80 error gyp info spawn make
80 error gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
80 error In file included from ../../nan/nan.h:176,
80 error                  from ../bindings/node/binding.cc:3:
80 error ../../nan/nan_callbacks.h:55:23: error: ‘AccessorSignature’ is not a member of ‘v8’
80 error    55 | typedef v8::Local<v8::AccessorSignature> Sig;
80 error       |                       ^~~~~~~~~~~~~~~~~
80 error ../../nan/nan_callbacks.h:55:40: error: template argument 1 is invalid
80 error    55 | typedef v8::Local<v8::AccessorSignature> Sig;
80 error       |                                        ^
80 error ../../nan/nan.h: In function ‘void Nan::SetAccessor(v8::Local<v8::ObjectTemplate>, v8::Local<v8::String>, GetterCallback, SetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, imp::Sig)’:
80 error ../../nan/nan.h:2542:19: error: no matching function for call to ‘v8::ObjectTemplate::SetAccessor(v8::Local<v8::String>&, void (*&)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Value>&), void (*&)(v8::Local<v8::Name>, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<void>&), v8::Local<v8::Object>&, v8::AccessControl&, v8::PropertyAttribute&, Nan::imp::Sig&)’
80 error  2542 |   tpl->SetAccessor(
80 error       |   ~~~~~~~~~~~~~~~~^
80 error  2543 |       name
80 error       |       ~~~~         
80 error  2544 |     , getter_
80 error       |     ~~~~~~~~~      
80 error  2545 |     , setter_
80 error       |     ~~~~~~~~~      
80 error  2546 |     , obj
80 error       |     ~~~~~          
80 error  2547 |     , settings
80 error       |     ~~~~~~~~~~     
80 error  2548 |     , attribute
80 error       |     ~~~~~~~~~~~    
80 error  2549 |     , signature);
80 error       |     ~~~~~~~~~~~~   
80 error In file included from /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-function.h:15,
80 error                  from /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8.h:33,
80 error                  from /home/imeyer/.cache/node-gyp/19.0.0/include/node/node.h:73,
80 error                  from ../bindings/node/binding.cc:2:
80 error /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-template.h:807:8: note: candidate: ‘void v8::ObjectTemplate::SetAccessor(v8::Local<v8::String>, v8::AccessorGetterCallback, v8::AccessorSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, v8::SideEffectType, v8::SideEffectType)’
80 error   807 |   void SetAccessor(
80 error       |        ^~~~~~~~~~~
80 error /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-template.h:812:22: note:   no known conversion for argument 7 from ‘Nan::imp::Sig’ {aka ‘int’} to ‘v8::SideEffectType’
80 error   812 |       SideEffectType getter_side_effect_type = SideEffectType::kHasSideEffect,
80 error       |       ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
80 error /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-template.h:814:8: note: candidate: ‘void v8::ObjectTemplate::SetAccessor(v8::Local<v8::Name>, v8::AccessorNameGetterCallback, v8::AccessorNameSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, v8::SideEffectType, v8::SideEffectType)’
80 error   814 |   void SetAccessor(
80 error       |        ^~~~~~~~~~~
80 error /home/imeyer/.cache/node-gyp/19.0.0/include/node/v8-template.h:819:22: note:   no known conversion for argument 7 from ‘Nan::imp::Sig’ {aka ‘int’} to ‘v8::SideEffectType’
80 error   819 |       SideEffectType getter_side_effect_type = SideEffectType::kHasSideEffect,
80 error       |       ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
80 error ../bindings/node/binding.cc: At global scope:
80 error /home/imeyer/.cache/node-gyp/19.0.0/include/node/node.h:976:7: warning: cast between incompatible function types from ‘void (*)(v8::Local<v8::Object>, v8::Local<v8::Object>)’ to ‘node::addon_register_func’ {aka ‘void (*)(v8::Local<v8::Object>, v8::Local<v8::Value>, void*)’} [-Wcast-function-type]
80 error   976 |       (node::addon_register_func) (regfunc),                          \
80 error       |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
80 error /home/imeyer/.cache/node-gyp/19.0.0/include/node/node.h:1010:3: note: in expansion of macro ‘NODE_MODULE_X’
80 error  1010 |   NODE_MODULE_X(modname, regfunc, NULL, 0)  // NOLINT (readability/null_usage)
80 error       |   ^~~~~~~~~~~~~
80 error ../bindings/node/binding.cc:26:1: note: in expansion of macro ‘NODE_MODULE’
80 error    26 | NODE_MODULE(tree_sitter_awk_binding, Init)
80 error       | ^~~~~~~~~~~
80 error make: *** [tree_sitter_awk_binding.target.mk:117: Release/obj.target/tree_sitter_awk_binding/bindings/node/binding.o] Error 1
80 error gyp ERR! build error 
80 error gyp ERR! stack Error: `make` failed with exit code: 2
80 error gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:203:23)
80 error gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
80 error gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:293:12)
80 error gyp ERR! System Linux 5.15.76-1-MANJARO
80 error gyp ERR! command "/usr/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
80 error gyp ERR! cwd /home/imeyer/node_test/node_modules/tree-sitter-awk
80 error gyp ERR! node -v v19.0.0
80 error gyp ERR! node-gyp -v v9.3.0
80 error gyp ERR! not ok
81 verbose exit 1
82 timing npm Completed in 5608ms
83 verbose unfinished npm timer reify 1668418964565
84 verbose unfinished npm timer reify:build 1668418966829
85 verbose unfinished npm timer build 1668418966829
86 verbose unfinished npm timer build:deps 1668418966829
87 verbose unfinished npm timer build:run:install 1668418966831
88 verbose unfinished npm timer build:run:install:node_modules/tree-sitter-awk 1668418966831
89 verbose code 1
90 error A complete log of this run can be found in:
90 error     /home/imeyer/.npm/_logs/2022-11-14T09_42_44_512Z-debug-0.log
```

**Comment by Beaglefoot** [comment]
I tested installation on node:19 docker image and this should be fixed in v0.5.2

**Comment by IngoMeyer441** [comment]
Thanks for the quick fix!
